yb-voyager/src/dbzm/valueConverter.go

```diff
@@ -186,19 +186,23 @@ func checkSourceExporter(exporterRole string) bool {
 	return exporterRole == "source_db_exporter"
 }
 
-func (conv *DebeziumValueConverter) convertMap(tableName string, m map[string]*string, exportSourceType string, formatIfRequired bool) error {
+func (conv *DebeziumValueConverter) convertMap(eventSchema string, tableName string, m map[string]*string, exportSourceType string, formatIfRequired bool) error {
 	var schemaRegistry *SchemaRegistry
+	targetTableName := tableName
```

Comment:
here `targetTableName` has nothing to do with the table names were using in the conversion, right?  It just for fetching the schema of the table.
If that's the case, maybe use something like `tableNameInSchemaRegistry` and add a small comment explaining the test case.